### PR TITLE
Add secrets.json example

### PR DESCRIPTION
**Pull request by alifjamaluddin**
Add secrets.json example and tell user to add it in gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,19 @@ for details. If you need to provide your own credentials, you can call
...
+	"SecretAccessKey":"your-secret-access-key"
+}
+```
+
+
+RAILS_ROOT/config/initializers/aws-sdk.rb
 ```ruby
 require 'json'
-# Because you're never going to commit credentials to source. Please.
+# Because you're never going to commit credentials to source. Please add it in .gitignore.
```
Comment (awood45) [inline README.md]: s/add it in/add it to

**Comment by awood45** [comment]
I'm happy to take this, and the test failure is (unsurprisingly) unrelated. One grammatical change and I'll pull this in. Thanks!

File: README.md (hunk 1 above)
Comment (anocriny) [inline README.md]: "Please add it into .gitignore"
This should be the right sentence is it?

Comment (awood45) [inline README.md]: Either works I think.

**Pull request closed without merging**